apps/vaporgui/FlowEventRouter.cpp

```diff
@@ -157,6 +167,25 @@ void FlowEventRouter::_updateTab()
     int numTS = GetActiveDataMgr()->GetNumTimeSteps();
     _pathlineLengthSlider->SetRange(0, std::max(1, numTS - 1));
     _pathlineInjectionSlider->SetRange(0, numTS);
+
+    std::vector<double> min, max;
+    FlowParams *        fp = dynamic_cast<FlowParams *>(GetActiveParams());
+    fp->GetBox()->GetExtents(min, max);
```

Comment:
I think it's encouraged to use the API with a fixed-size array when querying box extents. Check out this one:
```
void GetExtents(VAPoR::CoordType &minExt, VAPoR::CoordType &maxExt) const;
```

Reply:
Got it.  Fixed.